opm/grid/common/GridPartitioning.cpp

```diff
@@ -419,7 +452,8 @@ void addOverlapLayer(const CpGrid& grid, int index, const CpGrid::Codim<0>::Enti
             int index = ix.index(*it);
             auto owner = cell_part[index];
             exportProcs.insert(std::make_pair(owner, 0));
-            addOverlapLayer(grid, index, *it, owner, cell_part, exportList, layers-1);
+            //addOverlapLayer(grid, index, *it, owner, cell_part, exportList, layers-1);
+            addOverlapLayerNoZeroTrans(grid, index, owner, cell_part, exportList, layers-1, trans);
```

Comment:
What will happen if loadbalance is called without transmissibilities? Please add a test for this situation. Also: Please remove the old code rather than commenting it out.